CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ use patch releases for compatibility fixes instead.
 
 ## Unreleased
 
+### Changed
+
+- Increased the size of the connection pool
+
 ### Fixed
 
 - Fixed bug with `Beaker.(job|experiment).follow()` where final line was returned.
```

Comment:
Or at least word it in a way that's useful for other people, like:

```markdown
### Fixed

- Increased connection pool max size to avoid frequent warnings messages like this: `Connection pool is full, discarding connection: data.beaker.org....`
```